### PR TITLE
add bulk operations get path

### PR DESCRIPTION
**Pull request by chandlercarlile**
Issue: https://github.com/slingshotinc/web-app/issues/4348

This adds documentation for the bulk operations get endpoint. @slingshot-roybabbel and I worked together on #1, so this PR is pretty much a copy-pasta of that PR applied to the get endpoint.

### CODE REVIEW

File: openapi/components/responses/bulk_operations/get/get_success.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+description: OK
+content:
+  application/json:
+    schema:
+      type: object
+      properties:
+        data:
+          description: The bulk operation
```
Comment (kellyjensen) [inline openapi/components/responses/bulk_operations/get/get_success.yaml]: What is the bulk operation doing?  I'd add more detail.

Or is this supposed to be generic?

Comment (chandlercarlile) [reply openapi/components/responses/bulk_operations/get/get_success.yaml]: It's just referring to the Bulk Operation object that is being returned. I could make that more clear though!

File: openapi/paths/v1/bulk_operations@{uuid}.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  summary: Status
+  description: Check the status of a bulk operation job
+  operationId: bulkOperation
+  parameters:
+    - name: Company-Id
+      in: header
+      schema:
+        type: string
+        format: uuid
+
```
Comment (kellyjensen) [inline openapi/paths/v1/bulk_operations@{uuid}.yaml:13]: Remove empty line :-D

**Review by kellyjensen — APPROVED** [review]

File: openapi/openapi.yaml [hunk 3]
```diff
@@ -63,6 +63,8 @@ servers:
   - url: 'http://example.com/api/v1'
   - url: 'https://example.com/api/v1'
```
Comment (kellyjensen) [inline openapi/openapi.yaml:64]: Shouldn't this be `http://api.example.com/v1`?  Also shouldn't we use `getslingshot` instead of `example`?

Comment (chandlercarlile) [reply openapi/openapi.yaml:64]: yes, and yes.

Roy and I had categorized everything that wasn't specific to the bulk_operations endpoints we were working on as just general clean-up, and were going to put that into a separate issue. (which I have yet to make) So, that issue would be removing all the other example doc code, adding proper high level descriptions, fixing the logo, etc. We probably just bucketed this change under that, so thats why its still the example stuff.

**Pull request merged**